src/crypto/randomx/bytecode_machine.hpp

```diff
@@ -205,6 +206,14 @@ namespace randomx {
 			*ibc.fdst = rx_mul_vec_f128(*ibc.fdst, *ibc.fsrc);
 		}
 
+		static void exe_FMUL2I_R(RANDOMX_EXE_ARGS) {
+			 const rx_vec_f128 a = _mm_cvtepi32_pd(_mm_set_epi64x(0, *ibc.isrc)); 
+			*ibc.fdst = rx_mul_vec_f128(*ibc.fdst, a);
+			 const rx_vec_f128 b = _mm_cvtepi32_pd(_mm_set_epi64x(0, *ibc.isrc2));
+			*ibc.fdst = rx_mul_vec_f128(*ibc.fdst, b);
+			*ibc.fdst = rx_sqrt_vec_f128(*ibc.fdst);
```

Comment:
Either of `a` and `b` can be negative resulting in `sqrt(negative value)` here. This won't work reliably as a PoW.